exporting-charts/using-fusionexport/sdk-api-reference/java.md

```diff
@@ -196,137 +217,204 @@ exportConfig.set("templateHeight", 3000);
 ```
 
-#### `maxWaitForCaptureExit`
+### `orientation`
 
-Sets the maximum time FusionExport would wait for the CAPTURE_EXIT event to be triggered
+_**Type**: String_
 
-- **Type:** Integer
+This property will accept `portrait` or `landscape` as values. As the name suggests, it will set the orientation of the page. By default the value will be `portrait`.
 
-**Example**
+#### Example
+
+```javascript
+exportConfig.set('orientation', 'landscape');
+```
+
+### `pageMargin`
+
+_**Type**: String | Object_
+
+It will add margins on the page layout. It will accept both string and object. In case of string, margins will get applied on all four sides. In case of object `top`, `right`, `bottom`, `left` are the properties. Units for providing margins are:
+
+-  `px` - pixel
+-  `in` - inch
+-  `cm` - centimeter
+-  `mm` - millimeter
+
+#### Example
+
+```javascript
+exportConfig.set('margin', '10cm');
+exportConfig.set('margin', "{ top: '10px', right: '20px'}");
+```
+
+### `resourceFilePath`
+
+_**Type:** String_
+
+JSON file having the dependencies of the template when templateFilePath is provided. basePath denotes the base path of the project no local resource should be present outside this directory. include takes one or more glob to specify which files to send to the server. exclude take sone or more glob to specify which files should be excluded.
+
+#### Example
 
 ```java
-exportConfig.set("maxWaitForCaptureExit", 8000);
+exportConfig.set("resourceFilePath", "resources\resource.json");
 ```
 
-#### `type`
+The `resource.json` looks like as shown below:
 
-Sets the format of the output file. As of now it support `png`, `jpeg`, `svg` and `pdf`.
+```json
+{
+   "basePath": "../src/",
+   "include": ["**/*.js"],
+   "exlcude": [".env"]
+}
+```
 
-- **Type:** String
+### `callbackFilePath`
 
-**Example**
+_**Type:** String_
+
+Sets the path for a JavaScript file that would be injected at the bottom of the page for each export
+
+#### Example
 
 ```java
-exportConfig.set("type", "pdf");
+exportConfig.set("callbackFilePath", "resourcescallback.js");
 ```
 
-#### `quality`
+### `asyncCapture`
 
-Sets the quality of the output file. Provide either good, better or best
+_**Type:** Boolean_
 
-- **Type:** String
+Sets if the export process will wait for CAPTURE_EXIT event
 
-**Example**
+#### Example
 
 ```java
-exportConfig.set("quality", "best");
+exportConfig.set("asyncCapture", true);
 ```
 
-#### `outputFile`
+### `maxWaitForCaptureExit`
 
-Sets the output filename template, along with the path. You can write ejs style template for output file names. By default two functions are provided. number(start, end, interval) will resolve to a number respective to the position of the chart config in the chart config array in case of multiple file export. timestamp() will resolve to the current timestamp in unix format.
+_**Type:** Integer_
 
-- **Type:** String
+Sets the maximum time FusionExport would wait for the CAPTURE_EXIT event to be triggered
```

Comment:
Sets the maximum time that FusionExport waits for the CAPTURE_EXIT event to be triggered